themes/bootstrap5/templates/Auth/AbstractBase/resetpassword.phtml

```diff
@@ -0,0 +1,21 @@
+<?php if ('' !== ($username = $this->recoveryData['username'] ?? '')): ?>
+  <div class="form-group">
+    <label class="form-label"><?=$this->transEsc('Username') ?>:</label>
+    <p class="form-control-static"><?=$this->escapeHtml($username) ?></p>
+  </div>
+<?php endif; ?>
+<div class="form-group">
+  <label class="form-label" for="password"><?=$this->transEsc('new_password') ?>:</label>
+  <?php
+    $attrs = $this->htmlAttributes($this->passwordPolicy['inputAttrs'] ?? []);
+    $attrs->merge(['id' => 'password', 'type' => 'password', 'name' => 'password', 'class' => 'form-control', 'autocomplete' => 'new-password']);
+  ?>
+  <input <?=$attrs?> required>
```

Comment:
Do we need the space before the attributes, or will that result in double-space in the output?